.github/workflows/CI.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  build:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        Configuration: [Debug, Debug BMS, Debug 2013, Debug OE, Debug P2, Release, Release BMS, Release 2013, Release OE, Release P2]
+        include:
+          - Configuration: Release
+            SDK: orangebox
+            spt-filename: spt
+          - Configuration: Release BMS
+            SDK: bms
+            spt-filename: spt-bms
+          - Configuration: Release 2013
+            SDK: sdk2013
+            spt-filename: spt-2013
+          - Configuration: Release OE
+            SDK: episode1
+            spt-filename: spt-oe
+          - Configuration: Release P2
+            SDK: portal2
+            spt-filename: spt-p2
+          - Configuration: Debug
+            SDK: orangebox
+            spt-filename: spt
+          - Configuration: Debug BMS
+            SDK: bms
+            spt-filename: spt-bms
+          - Configuration: Debug 2013
+            SDK: sdk2013
+            spt-filename: spt-2013
+          - Configuration: Debug OE
+            SDK: episode1
+            spt-filename: spt-oe
+          - Configuration: Debug P2
+            SDK: portal2
+            spt-filename: spt-p2
+    steps:
+    - name: Checkout AlliedModders SDK
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Why? I read that the default is 1, which should be sufficient?

Reply:
Does that apply here or in general?

Reply:
In general.

Reply:
(as long as it works)